Lib/typemaps/cdata.swg

```diff
@@ -1,30 +1,246 @@
 /* -----------------------------------------------------------------------------
  * cdata.swg
  *
- * This library file contains macros for manipulating raw C data as strings.
+ * This library file contains macros for manipulating raw C data.
  * ----------------------------------------------------------------------------- */
 
+#ifdef SWIGGO
+#define swig_cdata_nelems_type int
+#else
+#define swig_cdata_nelems_type size_t
+#endif
+
```

Comment:
You've made lots of changes to make everything consistent, except for this Go. Why is that?

Reply:
Go is very sensitive with types.
`size_t` is defined as `int64`, probably to support large sizes (as `int` might be 32 on 32 bits CPUs).
`Lib/go/go.swg` line 233
`da5e02f81c (Ian Lance Taylor 2010-06-15 19:40:34 +0000 233) %typemap(gotype) size_t, const size_t & %{int64%}`

That will require in Go to use `Cdata(m, int64(7))`
 in `Examples/test-suite/go/li_cdata_runme.go`.

As string are not that long, I though it is an over kill.
My purpose is to make the "testing" consistent as possible.
As the testing represent the user-developer API.

Reply:
Now Go uses `int64`.
And all cdata uses `size_t`.